core/src/main/java/greencity/exception/handler/CustomExceptionHandler.java

```diff
@@ -745,4 +745,19 @@ public final ResponseEntity<Object> handleAccessDeniedException(AccessDeniedExce
         log.trace(ex.getMessage(), ex);
         return ResponseEntity.status(HttpStatus.FORBIDDEN).body(exceptionResponse);
     }
+
+    /**
+     * Method interceptor exception {@link TariffNotFoundException}.
+     *
+     * @param ex         Exception which should be intercepted.
+     * @param webRequest contain detail about occur exception.
+     * @return ResponseEntity which contain http status and body with message of
+     *         exception.
+     */
+    @ExceptionHandler(TariffNotFoundException.class)
+    public final ResponseEntity<Object> handleTariffNotFound(TariffNotFoundException ex, WebRequest webRequest) {
```

Comment:
we can define single handler for all *NotFound exceptions. Seems like handler logic is the same for all.